Code/Java/CircularQueue

```diff
@@ -0,0 +1,355 @@
+    Enter choice: 
+    3
+    Displaying the queue
+    14 50 -10 -78 45 
+    Do you wish to continue? Enter 1 for YES or 0 for NO: 
+    1
+    Choose from following: 
+    1. Insert an element
+    2. Delete an element
+    3. Display the elements: 
+    Enter choice: 
+    1
+    Enter element: 
+    60
+    Overflow
+    Do you wish to continue? Enter 1 for YES or 0 for NO: 
+    1
+    Choose from following: 
+    1. Insert an element
+    2. Delete an element
+    3. Display the elements: 
+    Enter choice: 
+    2
+    Deleted element: 14
+    Do you wish to continue? Enter 1 for YES or 0 for NO: 
+    1
+    Choose from following: 
+    1. Insert an element
+    2. Delete an element
+    3. Display the elements: 
+    Enter choice: 
+    3
+    Displaying the queue
+    50 -10 -78 45 
+    Do you wish to continue? Enter 1 for YES or 0 for NO: 
+    1
+    Choose from following: 
+    1. Insert an element
+    2. Delete an element
+    3. Display the elements: 
+    Enter choice: 
+    1
+    Enter element: 
+    60
+    Do you wish to continue? Enter 1 for YES or 0 for NO: 
+    1
+    Choose from following: 
+    1. Insert an element
+    2. Delete an element
+    3. Display the elements: 
+    Enter choice: 
+    3
+    Displaying the queue
+    50 -10 -78 45 60 
+    Do you wish to continue? Enter 1 for YES or 0 for NO: 
+    1
+    Choose from following: 
+    1. Insert an element
+    2. Delete an element
+    3. Display the elements: 
+    Enter choice: 
+    2
+    Deleted element: 50
+    Do you wish to continue? Enter 1 for YES or 0 for NO: 
+    1
+    Choose from following: 
+    1. Insert an element
+    2. Delete an element
+    3. Display the elements: 
+    Enter choice: 
+    2
+    Deleted element: -10
+    Do you wish to continue? Enter 1 for YES or 0 for NO: 
+    1
+    Choose from following: 
+    1. Insert an element
+    2. Delete an element
+    3. Display the elements: 
+    Enter choice: 
+    2
+    Deleted element: -78
+    Do you wish to continue? Enter 1 for YES or 0 for NO: 
+    1
+    Choose from following: 
+    1. Insert an element
+    2. Delete an element
+    3. Display the elements: 
+    Enter choice: 
+    2
+    Deleted element: 45
+    Do you wish to continue? Enter 1 for YES or 0 for NO: 
+    1
+    Choose from following: 
+    1. Insert an element
+    2. Delete an element
+    3. Display the elements: 
+    Enter choice: 
+    2
+    Deleted element: 60
+    Do you wish to continue? Enter 1 for YES or 0 for NO: 
+    1
+    Choose from following: 
+    1. Insert an element
+    2. Delete an element
+    3. Display the elements: 
+    Enter choice: 
+    2
+    Underflow
+    Do you wish to continue? Enter 1 for YES or 0 for NO: 
+    1
+    Choose from following: 
+    1. Insert an element
+    2. Delete an element
+    3. Display the elements: 
+    Enter choice: 
+    3
+    Circular Queue is Empty
+    Do you wish to continue? Enter 1 for YES or 0 for NO: 
+    0
+*/
```

Comment:
Merge these 2 multiline comments to one.

Reply:
Made the changes.